Action/Services/MediumService.cs

```diff
@@ -14,233 +15,266 @@
 using YamlDotNet.Serialization;
 using YamlDotNet.Serialization.NamingConventions;
 
-namespace PostMediumGitHubAction.Services
+namespace PostMediumGitHubAction.Services;
+
+public class MediumService : IMediumService
 {
-    public class MediumService
+    private readonly IConfigureService _configureService = new ConfigureService();
+    private readonly HttpClient _httpClient;
+    private Settings _settings;
+    private const string ApiMediumUrl = "https://api.medium.com/v1/";
+
+    public MediumService(Settings settings)
     {
-        private ConfigureService _configureService = new ConfigureService();
-        public async Task SubmitNewContentAsync()
+        _settings = settings;
```

Comment:
Why 2 constructors?
If needed, please invoke second constructor in this one.

Reply:
Perfect, thanks!